packages/devtools_app/lib/src/standalone_ui/vs_code/debug_sessions.dart

```diff
@@ -112,10 +117,18 @@ class _DevToolsMenu extends StatelessWidget {
         ? TextDirection.rtl
         : TextDirection.ltr;
 
-    Widget devToolsButton(
-      ScreenMetaData screen, {
-      bool enabled = true,
-    }) {
+    Widget? devToolsButton(Screen screen) {
+      // Don't include any screens that aren't appropriate.
+      if (!screen.requiresConnection ||
+          screen.requiresLibrary != null ||
+          screen.requiresVmDeveloperMode ||
+          screen.screenId == 'debugger') {
+        return null;
+      }
```

Comment:
Should we have a flag somewhere (like `ScreenMetadata` if we made the change above) for whether something should show up in the sidebar menu? It's not appropriate to show things like Home, Debugger.

Reply:
We're skipping those that need libraries for now (only provider) and will implement support for extensions in this menu by querying the server later.